packages/terra-alert/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated wdio screenshots due to examples changes.
+
```

Comment:
I believe wdio/jest tests normally aren't releaseable so we don't need a `Changed` entry for  wdio/test changes? This change also isn't directly associated with the `alert` component itself so it probably doesn't need this changelog. Someone can correct me if I'm wrong though.

Reply:
Good catch, removed a changelog for terra-alert: https://github.com/cerner/terra-core/pull/3905/commits/de37ab419d195da266b8a2c3efacbca353448573